lemur/plugins/lemur_aws/plugin.py

```diff
@@ -211,6 +298,12 @@ def get_certificates(self, options, **kwargs):
         ]
 
     def get_endpoints(self, options, **kwargs):
+        if self.get_option("endpointType", options) == "cloudfront":
```

Comment:
I like how this is written to support backwards compatibility, in case `endpointType` is None.